packages/cli/tests/utils/compiler.spec.ts

```diff
@@ -33,13 +33,23 @@ describe('Compiler utils', () => {
         test('should provide a valid build with block, settings and appBridge version', async () => {
             await compileBlock({ projectPath: rootPath, entryFile: pathToIndex, outputName: 'index' });
             await import(outputFile);
+
             expect(global.window).toHaveProperty('index');
             expect(global.window?.index.block).toBe('this is a block');
             expect(global.window?.index.settings).toMatchObject({ some: 'settings' });
             expect(global.window?.index.dependencies['@frontify/app-bridge']).toBe('^3.0.0-beta.99');
         });
     });
 
+    describe('compile Theme', () => {
```

Comment:
IMO we should also test whether the dependencies and settings are attached to the window. Wdyt?

Reply:
done